### ISSUE TITLE
Problem with amp-carousels generated by amp-list; user activated slideChange has "low trust"

### ISSUE THREAD

**Issue by joblog** [ampproject/amphtml#31856]
## What's the issue?

I have an amp-list that generates an amp-carousel for each item (using a mustache template)

I am trying to track each carousels last viewed slide by having each carousels code contain a on:slideChange that sets a corresponding state.
This is obviously triggered by the user pressing an arrow.
 
However when the user presses the arrow the state is not being set and the console error is similar to #24706
_[Action] "slideChange" event with "low" trust is not allowed to invoke "amp.setState"._

Significantly my page also contains another amp-carousel using the same source json (but not generated by an amp-list) and it is not experiencing any such problem. 
For this independent solitary carousel the slideChange event that occurs when a user presses an arrow seems to be a high trust event (as expected)

If this issue can be resolved please do so for amp-base-carousel as well

## How do we reproduce the issue?

Generate amp-carousels from an amp-list and examine if their user initiated slidechange events are "low trust"

## What browsers are affected?

All browsers

**Comment by micajuine-ho** [comment]
Hi @joblog thank you for you the issue report. Do you have a reproducible example that you can link or create in jsbin?

**Comment by joblog** [comment]
I'll get on that now.

**Comment by joblog** [comment]
OK I seem to have got to the bottom of this.

FWIW
I discovered I had an `<amp-action-micro> `which was closed off with a `</amp-bind-macro>` tag.

Fixing that fixed my problem.

My takeaway is that: 
I am running the official Wordpress AMP plugin. This (or something else) seemed to be fixing the above error on the fly (judging by page sourcecode). Consequently my AMP was passing all validators.

However - based on console messages - clicking an arrow in any of the carousels generated by my list was causing a refresh of every list on the page. 
This behaviour stopped when I corrected the above error.

Conclusion?: the automatic fixing of the error resulted in something akin to a page reload whenever an arrow was clicked, which led to the behaviour described in #24706
_[Action] "slideChange" event with "low" trust is not allowed to invoke "amp.setState"._
